README.md

```diff
@@ -169,7 +169,6 @@ A GitHub conflict is when people make changes to the same area or line in a file
 ---
 
 - ### **A**
-
```

Comment:
At line number 172, please re-enter a space after the letter A.

Let me know if you need any further assistance

Reply:
fine now?